sources/languages/php/description.md

```diff
@@ -0,0 +1,11 @@
+PHP (Hypertext Preprocessor) is a widely-used, open-source server-side scripting language designed for web development. It was created by Rasmus Lerdorf in 1994 and has since become one of the most popular languages for building dynamic web applications and websites.
```

Comment:
Thanks for including a reference, but it this page needs to refer to that reference:
```suggestion
According to [Wikipedia][1], PHP (Hypertext Preprocessor) is a widely-used, open-source server-side scripting language designed for web development. It was created by Rasmus Lerdorf in 1994 and has since become one of the most popular languages for building dynamic web applications and websites.
```